### PR TITLE
New Airlock script version

### LINKED ISSUE #19: Airlock V2 script
Scripts to manage airlock V2.

### PR DESCRIPTION
**Pull request by MaverickSora**
Resolve #19 

The SAS will be open if there is the same atmosphere on both sides.

### CODE REVIEW

File: Documentations/AdvancedAirlock.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- *STATUS_VENT_INWARD* : Wait with both doors closed
+
+The light is **On** in *STATUS_VENT_OUTWARD* and *STATUS_VENT_INWARD* status.
+
+The atmosphere is considered to be similar according to these constants :
+- DIFF_TEMPERATURE : Maximum temperature difference (default 10K)
+- DIFF_PRESSURE : Maximum pressure difference (default 10Kpa)
+- DIFF_OXYGEN : Maximum O2 ratio difference (default 0.02)
+- DIFF_CARBONDIOXIDE : Maximum CO2 ratio difference (default 0.01)
+- DIFF_VOLATILES : Maximum H2 ratio difference (default 0.01)
```
Comment (MaverickSora) [inline Documentations/AdvancedAirlock.md:48]: Where is Nitrogen ?

File: Documentations/AdvancedAirlock.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- D4 : Sensor In
+- D5 : Sensor Out
+
+This script update **Status Memory** to trigger the cycle.
+The status indicate which doors need to be *open/close*
+
+- *STATUS_VENT_OUTWARD* : Wait with both doors closed
+- *STATUS_WAIT_OUTWARD* : Wait trigger on inside door and open outside door
+- *STATUS_OPEN* : Open both doors
+- STATUS_WAIT_INWARD* : Wait trigger on outside door and open inside door
```
Comment (MaverickSora) [inline Documentations/AdvancedAirlock.md]: Missing *

**Comment by GothShoot** [comment]
What have changed ? 🤔

**Comment by MaverickSora** [comment]
> 
> 
> What have changed ? 🤔

Apart from the new script that keep airlock open if atmospher is similar ?
mmh... 🤔 
Just a breaking backward compatibility in devices list. 😅

**Review by GothShoot — APPROVED** [review]
Nice update ! 👍

**Pull request merged**